### ISSUE TITLE
Norway IDE Sites: 2017-2019

### ISSUE THREAD

**Issue by KW529** [felt0134/drought-net-tickets#147]
Norway sites include: Haveroya, Lygra (Young, Int., Old), Skotsvaer, Store Buoya

**Comment by KW529** [comment]
E-mail sent to PIs on 6-02-2020: 

Hi again,

Since more questions have emerged, I will list them all here (even ones sent in previous emails):
The Lygra sites (Young, Intermediate, Old) are missing a date for August in 2017. The current date reads, 2017-08-NA. Could you all please let me know the correct date?

1. Is it correct that you all decreased precipitation by 2% in the core drought treatments (see drought_trt column in the "sites" tab)?

2. Is your biomass data reported in g/m^2? It seems quite low for the amount of precipitation you all report receiving.

3. You all have different subplots within the same plot for biomass. Should we take an average of these values for that plot?

Thanks so much for any clarification!

Best,
Kate

**Comment by KW529** [comment]
Reply from Siri Haugum (6-02-2020):

"Hi Kate,
Thank you for notifying us about these errors! I hope they didn't cause much frustration.

I have answered each of your questions below, and attached an updataed data file + a picture clarifying question 4 (I am still not sure if I understand how if these should all be A or not, but I think you will know based on the attached picture).

1.  _The Lygra sites (Young, Intermediate, Old) are missing a date for August in 2017. The current date reads, 2017-08-NA. Could you all please let me know the correct date?_ Fixed!
2. _Is it correct that you all decreased precipitation by 2% in the core drought treatments (see drought_trt column in the "sites" tab)?_
I copied the fixed site information from last time I sent in the data until the new file that I downloaded from the IDE webpage, to be sure that I was using the most recent template for data sumbission. I was not aware that there was a new column in the site-sheet in the current template, so most of the site-spesific data was offset by one column. The '2' in the 'drought_trt ' column belonged to the 'drought_trt_record' column, etc. I have fixed this in the new version.
3. _Is your biomass data reported in g/m^2? It seems quite low for the amount of precipitation you all report receiving._ Fixed!
4. _You all have different subplots within the same plot for biomass. Should we take an average of these values for that plot?_ I am sorry, I think I have misunderstood how to report this, as I thought a "subplot" was one out of the four squares in our 2x2 meter plot. I have attached a picture explaining 'subplot A-D'.
![IMG-6162](https://user-images.githubusercontent.com/43038216/83656093-5cd1c300-a57c-11ea-9654-e41b75d1eb0f.JPG)
 I hope this make it clearer. I have not changed it all to 'A', but please do so if that is correct."

**Comment by KW529** [comment]
Listed in the IDE data folder and Raw csv folder as 'Norway_ide_data_2019' because all sites are together (Lygra, Haveroya, Skotsvaer, Store Buoya).

**Comment by KW529** [comment]
Sent 06-03-2020:

"Hi Siri,

Another question- sorry these arise at different stages of cleaning the data. I want to make sure that Lygra (Intermediate & Young), Haveroya, and Store Buoya do not have sampling dates in 2018. These sites only have sampling dates in 2017 and 2019. Thanks so much for any clarification! 

Best,
Kate"

**Comment by KW529** [comment]
Sent 6-03-2020: "P.S. Are the Ericales at your site woody or herbaceous plants?"

**Comment by KW529** [comment]
Response from Siri (06-04-2020):

"Please don't hesitate to ask. It's no trouble at all.
Yes, it is correct that Store Buøya, Haverøya and the young and intermediate sites on Lygra were only harvested in 2017 and 2019. We did not have enough resurces to harvest all sites in 2018, so we prioritized the old Lygra site and Skotsvær. We based that decision on the fact that these two are in a late successional phase, and hence less disturbed than the younger successional stages (our study system is Atlantic heathland which is managed by prescribed fire).

The Ericales are all woody. Calluna vulgaris is the most common, followed by Vaccinium spp. (vitis-idaea, myrtillus and uliginosum) and Empetrum nigrum.
The group "Wood" contains the other woody species in our system, which are Juniperus communis, and occasionally seedlings of Sorbus aucuparia, Betula pubescens, Betula nana and Salix spp.
It might make sense to merge Ericales and Wood. The evergreen Ericales are *much* more abundant than the two deciduous (V. myrtllus and uliginosum), and J. communis is *much* more abundant than the deciduous tree seedlings.

Because much of our biomass is dominated by these perannial woody species, we expect to see the strongest responses to the drought treatment amongs graminoids and forbs. Note that all our graminoids and forbs in the harvest are perannials, but only a very few species keep the aboveground shoot over winter.

Best,
Siri"

**Comment by KW529** [comment]
I followed up asking if woody species include secondary growth (06-09-2020).

**Comment by KW529** [comment]
Siri response (6-09-2020):

"Yes, it does. We have harvested all standing aboveground biomass destructively. Sadly, it was not possible for us to separate out annual growth on the perannial evergreen shrubs. I'm attaching two photos, one showing a site and another showing a plot. Calluna vulgaris is *very* dominant in most sites. Hope this make things clearer!
![IMG-9615](https://user-images.githubusercontent.com/43038216/84193657-08858200-aa59-11ea-9ad5-40a1fa1d2d2d.JPG)
![Uploading IMG-9632.JPG…]()

Best,
Siri"

**Comment by KW529** [comment]
I followed up again (06-09-2020) to make sure only Woody had secondary growth included. This is how Siri responded:

"Yes.

So:
Ericales (which is strictly wood): whole plant harvest
Wood (which is almost all evergreen and perennial Juniperus communis): whole plant harvest
Graminoids: separated into annual and previous growth (separated as 'Dead graminoids' for previous production and 'Graminoids' for annual production
Forbs: seperated into annual and previous growth in theory, but in practice it was all annual growth (perannial species which produce annual shoots)
Bryophytes: not separated into annual production. Almost all are mosses; the very few liverworths are probably too small to be picked up on the balance. Most species are pleurocarps. Dead bryophytes were sorted as litter.
Litter: mix of dead and loose plant material, mainly dead bryophytes and dead dwarf shoots of Calluna vulgaris."

**Comment by KW529** [comment]
Sent 6-17-2020:

"Hi Siri,

I have another question. In the treatment tab under the first_appl column (ie first treatment date), it states that the first treatment application was 2016-04-24. However, in the treatment notes column, it says that roofs were constructed between 2017-04-24 and 2017-05-01. Was the first drought treatment implemented in 2016 or 2017? Thanks so much for any clarification!

Best,
Kate"

P.S. Lygra Old has a 'mixed' category of taxa, what does this represent?"

I followed up with Siri on 6-29-2020:
"Hi Siri,

I hope all is well! I am checking in again to see if you had a chance to review my new questions, which I re-list below:
1. In the treatment tab under the first_appl column (ie first treatment date), it states that the first treatment application was 2016-04-24. However, in the treatment notes column, it says that roofs were constructed between 2017-04-24 and 2017-05-01. Was the first drought treatment implemented in 2016 or 2017?

2. One of the taxa categories at the Lygra Old is listed as 'mix'. What does this category mean?

Thanks so much for any clarification!"

**Comment by KW529** [comment]
Siri's reply (6-29-2020):

"1) This is possibly another misunderstanding of the which data should actually be entered. The first field work was done in 2016 (selecting plots, first round of registration of some response variables etc.), but the roofs were not installed before late April and early May 2017. I.e., the rainfall manipulation started in 2017.

2) I had to check this bag, and it contains Calluna, i.e. Ericales (we have numbered all the bags to make it easier to back trace the specific bag). So, it must be a typo. Very sorry about that. I also checked the two bags from plot 18.2 at SKO 2018. One contained a grass straw of a few centimetres and the leaf of a tiny forb, and the second contained two dwarf shoots of Calluna and one leaf of Vaccinium vitis-idea. The dry weight of both bags was neglectable."
